requirements/optional-public.txt

```diff
@@ -0,0 +1,7 @@
+git+https://github.com/cfpb/django-hud.git@1.2#egg=django-hud
+git+https://github.com/cfpb/owning-a-home-api.git@v0.9.7#egg=owning-a-home-api
+git+https://github.com/cfpb/retirement.git@0.5.0#egg=retirement
+git+https://github.com/cfpb/regulations-site.git@2.1.3#egg=regulations
+git+https://github.com/cfpb/regulations-core.git@1.2.3#egg=regcore
+git+https://github.com/cfpb/college-costs.git@2.2.2#egg=college-costs
+
```

Comment:
should add the public ccdb repo:

`git+https://github.com/cfpb/complaint.git@v1.2.3#egg=complaintdatabase`